proc_dash/utility.py

```diff
@@ -81,22 +81,23 @@ def extract_pipelines(bagel: pd.DataFrame, schema: str) -> dict:
     sort = bool(schema == "imaging")
 
     groupby = get_event_id_columns(bagel, schema)
+    # groupby auto-sorts keys (columns), but not actual column values
```

Comment:
why is this relevant? I'm assuming you __don't__ want the values sorted? In that case I'd say so explicitly, e.g. 
```suggestion
    # We only want to sort columns, not the values inside them. .groupby(sort=True) achieves this
```
might also consider renaming the column list to something other than `groupby` to avoid confusion